recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
 {% set name = "mamba" %}
-{% set libmamba_version = "2.0.3" %}
-{% set libmambapy_version = "2.0.3" %}
-{% set mamba_version = "2.0.3" %}
-{% set release = "2024.11.05" %}
+{% set libmamba_version = "2.0.4" %}
+{% set libmambapy_version = "2.0.4" %}
+{% set mamba_version = "2.0.4" %}
+{% set release = "2024.11.22-2" %}
```

Comment:
I just have seen that now, I am not really not comfortable with this release tag since this is inconsistent with previous ones but feel free to release it anyway.

Reply:
My reply was lost with the approval. So here it is again:
- nothing relies on this "date" release number, usually other packages depend on the version numbers of the "sub" packages. Only the build tools of the recipes may be affected when parsing the meta.yaml file, but the recipes has been built, so I don't see any issue in merging this.